lib/src/main/kotlin/io/bloco/faker/helpers/RandomHelper.kt

```diff
@@ -0,0 +1,76 @@
+package io.bloco.faker.helpers
+
+import java.security.SecureRandom
+import kotlin.math.abs
+import kotlin.math.pow
+
+class RandomHelper {
+    private val random: SecureRandom = SecureRandom()
+
+    fun <T> sample(options: List<T>): T {
+        return options[random.nextInt(options.size)]
+    }
+
+    fun digit(): String {
+        return random.nextInt(10).toString()
+    }
+
+    fun randBoolean(): Boolean {
```

Comment:
I think most of this class can be used as extensions and or use the kotlin random library:
like https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.random/-random/next-boolean.html